tests/shared/features/shared/session_steps.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Browser session steps
+
+  Background: Access some content in the main session
+    Given I go to "/static_pages/tab_1"
+    Then I should see "First browser tab"
+
+  Scenario: Browser sessions do not interfere with each other
```

Comment:
When you are checking multiple assertions in a single scenario, express the assertion as a comment.

In this case, I would group it sth. like this:
```
# Different pages can be accessed on different browser sessions
..
 I should see "second session" in the browser session "Second browser session"
...

# The default session is not affected by actions in other sessions
Then I should not see "Second browser session"
  And I should not see "Third browser session"

# Other sessions are not affected by the default session
...
````